pyomo/contrib/pynumero/sparse/tests/test_mpi_block_matrix.py

```diff
@@ -442,139 +442,6 @@ def test_sub(self):
         with self.assertRaises(Exception) as context:
             res = serial_mat2.tocoo() - mat1
 
-    def test_mul(self):
```

Comment:
Why are these tests being deleted?  Am I missing where the tests of this functionality are being re-added?

Reply:
All of the MPI dot product tests were moved to `test_mpi_block_matrix.py:TestMPIMatVec`.